hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/security/SecretKeyManagerService.java

```diff
@@ -158,7 +158,6 @@ public void stop() {
   }
 
   public static boolean isSecretKeyEnable(SecurityConfig conf) {
-    return conf.isSecurityEnabled() &&
-        (conf.isBlockTokenEnabled() || conf.isContainerTokenEnabled());
+    return conf.isSecurityEnabled();
```

Comment:
why is this changed?

Reply:
Before this patch, secretKey service only starts when block token or container token is enabled. With this patch, secretKey should start only when security is enabled, for delegation token is by default enabled when security is enabled.